NitroxServer/Player.cs

```diff
@@ -172,5 +175,29 @@ protected bool Equals(Player other)
         {
             return Id == other.Id;
         }
+
+        public void SetPingVisible(string pingKey, bool visibility)
+        {
+            // Default behaviour which we don't want to notice
+            if (visibility)
+            {
+                PingInstancePreferences.HiddenSignalPings.Remove(pingKey);
+                return;
+            }
+
+            PingInstancePreferences.HiddenSignalPings.Add(pingKey);
+        }
+
+        public void SetPingColor(string pingKey, int color)
+        {
+            // Default color which we don't want to notice
```

Comment:
What is the reason we don't want to notice when color is 0?

Reply:
because it will be set to 0 by default on client-side so it's just saving memory

Reply:
I would say that the memory you get here is not worth the complexity. I guess it's one of the reasons you don't use a Dictionary with a custom class or Tuple to save visibility and color. Imo you should switch to that.